src/main/java/frc/robot/Constants.java

```diff
@@ -21,10 +21,8 @@ public static final class ControllerPorts {
         public static final int OPERATOR_PORT = 1;
     }
 
-    /** PWM ports used by blinkin LED controllers */
-    public static final class BlinkinPorts {
-        public static final int SHOOTER_LEDS_PORT = 0;
-        public static final int INTAKE_LEDS_PORT = 1;
+    public static final class LEDPorts {
+        public static final int LED_PORT = 2;
```

Comment:
should be named in a way that indicates that candle uses a CAN id and not a port on the rio